runtime/doc/lsp.txt

```diff
@@ -116,6 +121,29 @@ To remove or override BUFFER-LOCAL defaults, define a |LspAttach| handler: >lua
       end,
     })
 <
+==============================================================================
+COMMANDS                                                    *:lsp*
+
+:lsp enable {name}?                                       *:lsp-enable*
+    Enables the given lsp clients. If no names are given, all clients
```

Comment:
Configs are "enabled", not "clients" or "servers". Servers are "started", and clients "attach".

No action required for now, just mentioning.